.github/ISSUE_TEMPLATE/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- (REQUIRED) What is the nature of this issue? -->
```

Comment:
I think we should consider using a bullet list or table format and use fewer headings to simplify and make more compact.

For example:

- Type of doc request:
    - [ ] New topic
    - [ ] Update to existing topic
    - [ ] Other
- Magento 2 GitHub issue:
- Magento 2 GitHub PR: 
- Magento 2 product: <!--  Is this really necessary? Aren't all contributions submitted to Open Source and then ported to Commerce by internal staff later, if at all? -->
    - [ ] Open Source
    - [ ] Commerce
- Magento 2 version:
   - [ ] 2.1
   - [ ] 2.2
   - [ ] 2.3

OR

I'm not sure if the checkboxes formatting works in a table... but you get the idea.

| Item                   | Description                                                |
|------------------------|------------------------------------------------------------|
| Type of doc request    | - [ ] New topic - [ ] Update to existing topic - [ ] Other |
| Magento 2 GitHub issue | [link](link)                                                     |
| Magento 2 GitHub PR    | [link](link)                                                     |
| Magento 2 product      | - [ ] Open Source - [ ] Commerce                           |
| Magento 2 version      | - [ ] 2.1 - [ ] 2.2 - [ ] 2.3                              |

## Content
<!-- (REQUIRED) Use this are to provide proposed doc updates -->

Reply:
I definitely like these formatting ideas. Reformatting content.